Makefile

```diff
@@ -45,6 +46,13 @@ build-linux: ; $(info building executable for Linux x86_64…) @ ## Build progra
 		-ldflags '-X $(MODULE)/internal/config.Version=$(VERSION) -X $(MODULE)/internal/config.BuildDate=$(DATE)' \
 		-o $(BIN_DIR)/$(BIN_LINUX) cmd/$(CLI)/main.go
 
+.PHONY: build-dockerised
```

Comment:
why a new binary, what is different from the one above?

Reply:
`CGO_ENABLED=0` seems to be the only difference, totally new flag for me 🤔 Seems to control the CGo standard libraries (build them vs use prebuilt) or something(?)

Reply:
CGO_ENABLED=0 and especially -w for LD_FLAGS. Unfortunately the binary won't run under scratch without them.

Reply:
Now with a longer answer as I have the time. CGO_ENABLED=0 creates a statically linked binary, that is needed as the `scratch` container doesn't have any existing libraries. It leads to a bigger binary (10MB vs ~3MB), but it's still a lot smaller than having those in an Alpine container. -w in ldflags turns off debugging information, leading to a smaller binary than what we'd otherwise get.